docs/cryoet_data_portal_docsite_landing.md

```diff
@@ -1,13 +1,8 @@
-# CryoET Data Portal
+# Browse and visualize datasets in the new beta site: https://cryoetdataportal.czscience.com
 
-The Chan Zuckerberg Institute for Advanced Biological Imaging (CZ Imaging Institute) has developed the prototype CryoET Data Portal providing queryable and organized data from cryoET experiments. The initial target users are segmentation algorithm developers to produce annotations for diverse macromolecules in the tomogram that may be used for high-resolution subtomogram averaging. We currently have 120 tomograms from three datasets contributed by the groups of Julia Mahamid and Jürgen Plitzko. Each of these tomograms have a minimum of “ground truth” point annotations of ribosomes. We also provide a Python API to query and download the data. This portal is not intended to be a final product, but rather, a first step to start a discussion in the wider community as to what is needed with such a portal. We welcome feedback from the community on the data structure, design and functionality. Once we have gathered some feedback we plan to organize a community-wide workshop to discuss next steps.
+# Read documentation and API  in \*this* alpha site:<br>CryoET Data Portal
```

Comment:
Is the word `this` or `CryoET Data Portal` was supposed to link to the alpha site?

Reply:
No, I'm just trying to figure out how best to convey that the site they're on is the documentation site. If you have ideas, happy to hear!

Reply:
That makes sense. The break tag threw me off when I was reading the md file.